api/types/types.go

```diff
@@ -119,6 +119,11 @@ var ControllerPolicyRule = []rbacv1.PolicyRule{
 		APIGroups: []string{"networking.k8s.io"},
 		Resources: []string{"ingresses"},
 	},
+	{
```

Comment:
It would be nice to not require this permission (in which case of course events would not be published this way). Not sure what is the simplest way to do that though... could perhaps try to a create a separate role and then  if that succeeds add a role binding for that (and if it fails just disable events).

Reply:
Updated!